packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -224,6 +233,15 @@ cp bin/* %{buildroot}%{_sbindir}/
 cp extra/katello-tracer-upload-dnf %{buildroot}%{_sbindir}/katello-tracer-upload
 %endif
 
+%if 0%{?suse_version} >= 1500
+sed -i 's/bin\/python$/bin\/python3/' %{buildroot}%{_sbindir}/katello-enabled-repos-upload
```

Comment:
It's merged so no need to modify it, but for future reference, you can use `s|bin/python$|bin/python3|` with sed. Saves escaping and makes it more readable.

Reply:
Thanks. Nice improvement.
See: https://github.com/theforeman/foreman-packaging/pull/7500